### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by Amirsil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
 
+For this example ill be using Management Services, which is a logical group of services that are deployed on management hubs (Previously known as Management Hub Components).
+
+2 Repos will coexist:
+- Preprod Management Service Set
+- Prod Management Service Set
+
+ServiceSet - A repo template for Argo ApplicationSet which deploys a set of services defined in values.yaml to a specific cluster.
```
Comment (DavidDaskalo) [inline README.md]: What is ApplicationSet? Give some insight to the reader
Do not assume they know everything, at the very least place a link to argo docs for explanation

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+A service is a helm chart reference defined by:
+- serviceName
+- repoURL
+- targetRevision
+- path
+
+Lets say I want to add a feature to a project that is a part of Management Service Set (After application CI has passed successfully).
+I'll create a new branch for the feature in the helm chart and make the desired changes.
```
Comment (DavidDaskalo) [inline README.md]: Maybe add some example to ensure that everything is more clear for the reader

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+Then, to check the project in a preprod environment, I will change the targetRevision of the desired service reference in "Preprod Management Service" Set's values.yaml, and sync `Preprod Management Service Set` in Argo. 
+
+Lets say everything went well and the application is behaving expectedly and integrating well with its environment.
+Now we want to deploy it to production.
+
+All we have to do now is to create a merge request in the stand-alone helm chart from the branch we worked on to `master` (Preferebly squash commits to avoid `Updated *.yaml` garbage commits and give the single commit a meaningful name).
```
Comment (DavidDaskalo) [inline README.md]: One exception is when the commits are meaningful and should not be squashed

File: README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+- Preprod Management Service Set
+- Prod Management Service Set
+
+ServiceSet - A repo template for Argo ApplicationSet which deploys a set of services defined in values.yaml to a specific cluster.
+
+A service is a helm chart reference defined by:
+- serviceName
+- repoURL
+- targetRevision
+- path
```
Comment (DavidDaskalo) [inline README.md]: Add code snippet or template example for clarity, this can be interpreted by readers with little knowledge in multiple ways (ambigious)

**Review by DavidDaskalo — CHANGES_REQUESTED** [review]
All in all this looks solid, but you should elaborate the doc to be a little more generic and add some examples and snippets to ensure clarity for the readers.

A good doc is a doc that even an unskilled reader can interpret and understand!
Especially if it has diagrams/drawings and etc

File: README.md (hunk 1 above)
Comment (Amirsil) [reply README.md]: Just added a brief explanation and a link to read more about ApplicationSet

File: README.md (hunk 4 above)
Comment (Amirsil) [reply README.md]: Added `services` example

File: README.md (hunk 3 above)
Comment (Amirsil) [reply README.md]: Right, but usually with yamls commits are usually something like `Updated *.yaml`

File: README.md (hunk 2 above)
Comment (Amirsil) [reply README.md]: 👍

**Review by DavidDaskalo — APPROVED** [review]

**Review by DavidDaskalo — CHANGES_REQUESTED** [review]
This doc displays exclusively the idea you worked on , for it to be really an ADR like doc you need to review and compare with regard to the basis on which the idea was built and based on.

Imagine an objective reader, this doc will show only one approach, I reference you to this docs for further understanding of what I mean.

- https://github.com/joelparkerhenderson/architecture-decision-record
- https://adr.github.io/

**Pull request merged**